src/main/java/net/snowflake/client/core/SFSessionProperty.java

```diff
@@ -57,8 +58,8 @@ public enum SFSessionProperty {
   PRIVATE_KEY_FILE_PWD("private_key_file_pwd", false, String.class),
   CLIENT_INFO("snowflakeClientInfo", false, String.class),
   ALLOW_UNDERSCORES_IN_HOST("allowUnderscoresInHost", false, Boolean.class),
-  CLIENT_OUT_OF_BAND_TELEMETRY_ENABLED(
-      "CLIENT_OUT_OF_BAND_TELEMETRY_ENABLED", false, Boolean.class),
+  HTAP_TELEMETRY_ENABLED(
```

Comment:
What are chances HTAP also have OOB telemetry? if this param is for HTAP OOB then please rename variables to reflect that.

Reply:
@sfc-gh-igarish this is a PR to make a separate parameter for controlling HTAP-related OOB telemetry. Are you saying you would like OOB to be also included in the name of the parameter?

Reply:
@sfc-gh-mknister Yes Meg. Better to add OOB in the name for HTAP.